protocol/specification/descriptive.rst

```diff
@@ -247,6 +306,10 @@ Optional Parameter Properties
 
     The value given here must conform to the data type of the accessible.
 
+.. _parameter-meaning:
+``"meaning"``
+   A dictionary regarding the parameter meaning. It has the same specification as the :ref:`module-meaning <module meaning>`.
```

Comment:
Is this reference defined?

Reply:
ah thanks i think this one got lost when i recreated the branch